Se agregó modelo para las compañías

Jesus-Vila opened the pull request:
Se agregó el modelo para representar a las compañías y una serie de pruebas

vila8 committed:
Add user profile

vila8 committed:
Add new line at the end of file

vila8 committed:
Refactore user validations

Refactor user and user profile validations
Add email field to user

vila8 committed:
Refactor user serializer

Change user profile fields at same level

vila8 committed:
Change user profile serializer

Add inheritance to user profile

vila8 committed:
Add company

Add company model and tests

vila8 committed:
Add company

Add company model and tests

vila8 committed:
Add company

Add company model and tests

vila8 committed:
Add company

Add company model and tests

kyokun commented on products/api/fixtures/company.json:
```diff
@@ -0,0 +1,65 @@
+[
```
Este fixture, como lo estás usando para pruebas no debe ir aqui. Cambialo a tests/fixtures/.
La idea es que en esta carpeta sólo vaya aquella información necesaria para empezar a usar la aplicación.

kyokun commented on products/api/serializers/company.py:
```diff
@@ -0,0 +1,45 @@
+
+class AddressSerializer(serializers.Serializer):
+    """Address serializer."""
+    street = serializers.CharField(max_length=100)
```
Aqui no es necesario que declares los campos explictamente (ya que tienen el mismo nombre que en el modelo) Podrías usar:

```
class Meta:
        model = Company
        fields = (
            'street',
            'country',
            'num_ext',
            'zip_code',
            'state',
        )
```

vila8 committed:
Add product models

Add product model
Add product attribute model
Add product extra information model
Add product logistic model

vila8 committed:
Add product models

Add product model
Add product attribute model
Add product extra information model
Add product logistic model

vila8 committed:
Refactor address serializer

Move company fixtures to tests directory

vila8 committed:
Refactor address serializer

Move company fixtures to tests directory

vila8 committed:
Refactor address serializer

Move company fixtures to tests directory

vila8 committed:
Refactor address serializer

Move company fixtures to tests directory

vila8 committed:
Add new line at the end of file

vila8 committed:
Add new line at the end of file

vila8 committed:
Refactor user model and serializer

Use inheritance to create user profiles

vila8 committed:
Follow style guide for python code

vila8 committed:
Follow style guide for python code

vila8 committed:
Follow style guide for python code

vila8 committed:
Follow style guide for python code

vila8 committed:
Remove unnecessary Model.save() and view

vila8 committed:
Remove unnecessary Model.save() and view

vila8 committed:
Change extra info product

Change values assignation

vila8 committed:
Change extra info product

Change values assignation

kyokun committed:
Merge pull request #4 from Jesus-Vila/model_users

Se agregaron modelos para los usuarios

vila8 committed:
Refactor product serializers

vila8 committed:
Fix merge conflict

vila8 committed:
Merge branch 'models_product' of github.com:Jesus-Vila/DjangoTests into models_product

Fix merge conflict

vila8 committed:
Merge branch 'models_company' of github.com:Jesus-Vila/DjangoTests into models_company

Fix merge conflicts

kyokun committed:
Merge pull request #6 from Jesus-Vila/models_product

Se agregaron modelos para los productos

vila8 committed:
Move fixtures files

vila8 committed:
Merge branch 'models_company' of github.com:Jesus-Vila/DjangoTests into models_company

vila8 committed:
Move fixtures files
